plugins/kinet/KiNetNode.cpp

```diff
@@ -136,9 +142,9 @@ bool KiNetNode::SendPortOut(const IPV4Address &target_ip,
 
   m_output_queue.Clear();
   PopulatePacketHeader(KINET_PORTOUT_MSG);
-  m_output_stream << universe << port << padding
-                  << flags << padding  // Are the flags actually 16 bit?
-                  << static_cast<uint16_t>(buffer.Size());
+  m_output_stream << universe << port
+                  << flags1 << flags2
+                  << HostToBigEndian(static_cast<uint16_t>(buffer.Size())); // Buffer must be at least 24 bytes
```

Comment:
You can send a smaller packet with OLA like so:
`./examples/ola_streaming_client -u 2 --dmx 1,2,3,4`

The website only sends a full frame of 512 slots each time.

Reply:
OK, I will add some code to pad with zeros if the buffer is smaller.

This comes from the QuickPlay Pro 1 options panel:
![image](https://user-images.githubusercontent.com/4472083/190932351-48cd47e8-1f8d-4b82-b0c3-3ceac8d4288b.png)

Reply:
OK, I am now actually enforcing the minimum size. See 68d6dfe.

Reply:
> This comes from the QuickPlay Pro 1 options panel: ![image](https://user-images.githubusercontent.com/4472083/190932351-48cd47e8-1f8d-4b82-b0c3-3ceac8d4288b.png)

So can you set the range in QPP1 to fewer than 24 channels? If so does it pad in QPP1, or in the PSU?